README.md

```diff
@@ -97,6 +97,26 @@ make
 Note: by using the `-G` option of `cmake` you can specify a generator, e.g. `cmake -G Xcode -DCMAKE_BUILD_TYPE=Release ..` will generate an Xcode project.
 Please check `cmake --help` for more options.
 
+### Especially macOS
+
+```bash
+git clone --recursive https://github.com/Return-To-The-Roots/s25client s25client
+cd s25client
+mkdir -p build && cd build
+cmake .. \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_OSX_ARCHITECTURES=arm64 \
```

Comment:
Why doesn't it work without that? We have [code](https://github.com/Return-To-The-Roots/s25client/blob/2d68497723d0c70db410cb7f945d881821cb4f9e/cmake/darwin.cmake) that should ensure it works out of the box